utmap-server/src/app/main/__init__.py

```diff
@@ -1,8 +1,10 @@
 from flask import Flask
 from pymongo import MongoClient
-from .config import configByName, dbHost, dbPort, dbName, serverHost, serverPort, debugSetting, skipDBInit
+from .config import configByName, dbHost, dbPort, dbUser, dbPassword, dbName, serverHost, serverPort, debugSetting, skipDBInit
 
-db = MongoClient(dbHost, dbPort).get_database(dbName)
+db_connect_string = "mongodb+srv://{user}:{password}@cluster0.slrlz.mongodb.net/UTMap?retryWrites=true&w=majority".format(user=dbUser, password=dbPassword)
```

Comment:
This is a secret, its passed in as an environment variable. This file imports `dbHost` at the top.